model/clusters_mgmt/v1/pagination_type.model

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright (c) 2024 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Pagination contains the data which enables pagination on the response
+struct Pagination {
```

Comment:
Not that big a fan of this as it is specific only for one model. 
How do other models have the pagination implemented?

Reply:
BreakGlassCredentialsListRequest has implemented pagination the request struct like this: https://github.com/openshift-online/ocm-sdk-go/blob/d6662dfa3e339336e2459a413289c5b867500131/clustersmgmt/v1/break_glass_credentials_client.go#L350
I think this way it's more pluggable to any struct that needs to implement pagination on the response.